packages/js/test/e2e/global-setup.ts

```diff
@@ -0,0 +1,30 @@
+import { bsLocal, BS_LOCAL_ARGS } from './browserstack.config'
+import { promisify } from 'util'
+
+const sleep = promisify(setTimeout)
+const redColour = '\x1b[31m'
+const whiteColour = '\x1b[0m'
+module.exports = async () => {
+  if (!process.env.BROWSERSTACK_ACCESS_KEY) {
+    console.log('Will not start BrowserStackLocal because BROWSERSTACK_ACCESS_KEY is not set')
+
+    return
+  }
+
+  console.log('Starting BrowserStackLocal ...');
+  // Starts the Local instance with the required arguments
+  let localResponseReceived = false;
+  bsLocal.start(BS_LOCAL_ARGS, (err) => {
+    if (err) {
+      console.error(
+        `${redColour}Error starting BrowserStackLocal${whiteColour}`
```

Comment:
Can you log out this error? I got this message but it didn't come with any actionable information.

Reply:
With "log out", you mean to log the `err` itself, right? Done!